# Katz Penfold Calculations

This section will perform the calculations to determine the maximum range of the high energy beta particles emitted from the radioactive source.
The initial source is Sr-90, which has a maximum emission of roughly 0.546 MeV, however, its beta daughter, Y-90, has a maximum emission of 2.28 MeV. Since our goal is to limit all of these beta particles, we will calculate based on that absolute maximum. 

For low-medium energies (0.0-2.5 MeV), the equation is defined as


**$R = 412 \cdot E^{(1.265 - 0.0954 \ln E)}$** 

R is $mg/cm^2$ and E is MeV




In [1]:
import math
#code to compute R where E is 2.28 MeV
E = 2.28  # Energy in mega electron volts (MeV)
# Constants

def CalculateKatzPenfold(energy):
    return 412 * energy ** (1.265 - 0.0954 * math.log(energy))

print("Katz-Penfold R value for E =", E, "MeV is:", CalculateKatzPenfold(E), "mg/cm^2")

Katz-Penfold R value for E = 2.28 MeV is: 1095.321699239395 mg/cm^2


Now having determined R, we must convert to grams, and then divide by the density of the material which the particles will be passing through.
In this case, the material is water, so the density is $1.0 g/cm^3$

In [2]:
#code to convert R (mg/cm^2) to R (cm) using density of water (1 g/cm^3)
def ConvertRToCm(R_mg_cm2, density_g_cm3=1.0):
    return R_mg_cm2 / (density_g_cm3 * 1000)  # Convert mg/cm^2 to g/cm^2 and then to cm

R_cm = ConvertRToCm(CalculateKatzPenfold(E))
print("Katz-Penfold R value for E =", E, "MeV is:", R_cm, "cm")

Katz-Penfold R value for E = 2.28 MeV is: 1.095321699239395 cm


This gives R ≈ 1.1 cm. The result is precise only to ±5-10%, so it will be rounded to 1.10~ cm.
## 1.10~ cm